### PR TITLE
DEVELOPER-1273 Add the DV 6.1 GA download to replace the Beta download

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

**Comment by paulrobinson** [comment]
You also need to update:

    current_version: 6.0.0.GA

To:

    current_version: 6.1.0.GA

**Comment by Dantheman720** [comment]
Arg!  I overwrote that in the diff compare…will fix and resubmit

File: products/datavirt/_common/product.yml [hunk 1]
```diff
@@ -26,6 +26,14 @@ guides:
 description: 'A data virtualization tool useful for any organization struggling to gain operational and analytical insights from data dispersed and locked up in various business units, applications, and technology silos. (formerly known as JBoss Enterprise Data Services Platform).'
 default_download_artifact_type: installer
 downloads:
+  6.1.0.GA:
+    release_date: 2015-03-11
+    description: Minor release of JBoss DV 6.x series
```
Comment (paulrobinson) [inline products/datavirt/_common/product.yml]: Kenny requested that you use the description from the Beta. So:

    Improved User Interface, OpenShift Beta Cartridge, Kerberos, Audit Log in Dashboard, MongoDB, Cloudera/Impala. Apache Solr, EAP 6.3, JDG 6.2.

**Comment by paulrobinson** [comment]
Also, make sure the documentation links on the resources page work

**Pull request merged**